### PR TITLE
CORCI-812 release: Support releases from branches

### PR DESCRIPTION
**Pull request by brianjmurrell**
Also handle pre-releases which are any release with a - in it.
- I.e. 1.2.3-alpha1, 1.2.3-beta1, 1.2.3-rc1, etc.

Allow setting additional variables such as project and remote names.
- mostly for testing.

Fix CORCI-832 AttributeError: 'module' object has no attribute 'AlreadyExistsError'

Skip-build: true
Skip-test: true
Skip-run_tests: true
Signed-off-by: Brian J. Murrell <brian.murrell@intel.com>

### CODE REVIEW

File: .github/workflows/crease_release.yml [hunk 1]
```diff
@@ -4,6 +4,7 @@ on:
   push:
     branches:
     - master
+    - 'releases/**'
```
Comment (jolivier23) [inline .github/workflows/crease_release.yml]: is there a reason this file is named crease_release?

**Comment by brianjmurrell** [comment]
> is there a reason this file is named crease_release? 

Lol.  A typo.

File: .github/workflows/crease_release.yml (hunk 1 above)
Comment (brianjmurrell) [reply .github/workflows/crease_release.yml]: Lol. A typo. Let me push a fix, once I have tested it.

File: SConstruct [hunk 2]
```diff
@@ -109,6 +125,42 @@ def scons():
...
+        variables.Add('REMOTE_NAME', 'The remoten name release on.', 'origin')
+
+        env = Environment(variables=variables)
+
+        org_name = env['ORG_NAME']
+        remote_name = env['REMOTE_NAME']
+        base_branch = env['RELEASE_BASE']
+
+        try:
+            tag = env['RELEASE']
```
Comment (jolivier23) [inline SConstruct]: Does this actually throw an exception?  the default should be None

Comment (brianjmurrell) [reply SConstruct]: It must:

```bash
$ scons release
scons: Reading SConscript files ...
Usage: scons RELEASE=x.y.z release
```

We can only get that error message if the exception is thrown.

**Pull request merged**